src/serverModuleGen.py

```diff
@@ -22,6 +25,21 @@ def generateServerIncludes(config_filename, reg_filename):
 
     serverfile.write("}\n")
 
+def generateSearchPaths(filename):
+    with open(filename) as configfile:
+        modules = configfile.readlines()
+        ret = ""
+        for module in modules:
+            module = (module.rstrip()).lstrip()
```

Comment:
I think you can simplify this as `module = module.strip()`  which should trim whitespace from both ends.